src/bot.js

```diff
@@ -297,7 +297,7 @@ client.on("threadCreate", async (post) => {
   const forumTags = [];
 
   // Send message
-  const allowedChannels = []; // @Waren
+  const allowedChannels = []; // Only respond in appropriate channels
```

Comment:
Hey @kien-ngo, kindly reference this back to the config file. Create a new entry there for this. The value should be in the config. This is for easy management. We don't need to change the code just to change the settings.

Reply:
@warengonzaga I'm going to remove this change so we can get this merged to "Dev"

Reply:
I think I can handle it when it is available in the Dev branch.